spec.html

```diff
@@ -13193,24 +13416,24 @@ <h1>ECMAScript Function Objects</h1>
         </tr>
         <tr>
           <td>
-            [[Fields]]
+            [[Elements]]
           </td>
           <td>
-            List of ClassFieldDefinition Records
+            List of ClassElementDefinition Records
           </td>
           <td>
-            If the function is a class, this is a list of Records representing the non-static fields and corresponding initializers of the class.
+            If the function is a class, this is a list of Records representing the non-static elements and corresponding initializers of the class.
           </td>
         </tr>
         <tr>
           <td>
-            [[PrivateMethods]]
+            [[Initializers]]
           </td>
           <td>
-            List of PrivateElements
+            List of function objects | ~empty~
           </td>
           <td>
-            If the function is a class, this is a list representing the non-static private methods and accessors of the class.
+            If the function is a class with decorated prototype methods or accessors, this is a list of initializer functions added by the decorators, or ~empty~.
```

Comment:
could this also apply to a class with a class-level decoration, but no decorated prototype methods or accessors? or, a class with only field decorators?